src/ceph-disk/ceph_disk/main.py

```diff
@@ -56,6 +56,16 @@
             'ready': 'cafecafe-9b03-4f30-b4c6-b4b80ceff106',
             'tobe': 'cafecafe-9b03-4f30-b4c6-b4b80ceff106',
         },
+        'block.db': {
+            # identical because creating a block is atomic
+            'ready': '30cd0809-c2b2-499c-8879-2d6b78529876',
+            'tobe': '30cd0809-c2b2-499c-8879-2d6b78529876',
```

Comment:
s/d28/2be/ because it's cute and convenient to quickly figure out what is tobe (2be)... :-)